cpp/test/Slice/errorDetection/IllegalIdentifier.ice

```diff
@@ -1,20 +1,8 @@
-
-// NOTE: Keep the initial blank line. This is necessary to avoid a problem
-// with system-provided mcpp versions that don't include ZeroC's BOM patch.
```

Comment:
This comment is from 7 years ago. So I guess this has been a known issue for a while... And we even knew that having a comment on the first line triggered the re-ordering bug.

Adding this blank line at the top feels like a lame fix though.